src/Tmds.Ssh/ChannelContextSendMessageExtensions.cs

```diff
@@ -182,5 +182,55 @@ byte      SSH_MSG_CHANNEL_WINDOW_ADJUST
                 return packet.Move();
             }
         }
+
+        public static ValueTask StartSftpAsync(this ChannelContext context, CancellationToken ct)
```

Comment:
Rename to `SendChannelSubsystemMessageAsync` and accept the subsystem as a string.